appinfo/info.xml

```diff
@@ -24,7 +24,7 @@ This extension provides the most simple way to add a second factor to user login
 	</settings>
 
 	<dependencies>
-		<owncloud min-version="10.0" max-version="10.0" />
+		<owncloud min-version="10.0" max-version="10.1" />
 		<php min-version="5.4" max-version="7.2" />
```

Comment:
Is min version of php 5.4 still valid?

Reply:
I think we should have the min-version of PHP set to what we are testing, I've changed it to 5.6